tests/brevitas_examples/test_llm.py

```diff
@@ -0,0 +1,442 @@
+    args.export_prefix = ptid2pathname(request.node.nodeid)
+    args.weight_bit_width = 8
+    args.weight_quant_granularity = "per_channel"  # "per_tensor", "per_channel", "per_group".
+    args.input_bit_width = 8
+    args.act_calibration = True
+    return args
+
+
+@pytest.fixture(
+    params=[
+        {},
+        {
+            "bias_corr": True},
+        {
+            "act_equalization": "layerwise"},
+        {
+            "act_equalization": "fx"},
+        {
+            "weight_equalization": True},
+        {
+            "gptq": True},
+        {
+            "ln_affine_merge": True},])
+def toggle_run_args(default_run_args, request):
+    args = default_run_args
+    args.update(**request.param)
+    yield args
+
+
+@pytest.mark.llm
+@requires_pt_ge('2.2')
+def test_small_models_toggle_run_args(caplog, toggle_run_args, small_models_with_ppl):
+    caplog.set_level(logging.INFO)
+    args = toggle_run_args
+    args.model = small_models_with_ppl.name
+    exp_float_ppl = small_models_with_ppl.float_ppl
+    use_fx = requires_fx(args)
+    if use_fx and not small_models_with_ppl.supports_fx:
+        pytest.xfail(f"{small_models_with_ppl.name} does not support FX")
+    float_ppl, quant_ppl, model = main(args)
+
+
+@pytest.fixture(
+    scope="session",
+    params=[
+        ModelAndPpl(
+            name="hf-internal-testing/tiny-random-OPTForCausalLM",
+            float_ppl=None,
+            supports_fx=True,
+        ),])
+def small_models_with_ppl_pt_ge_2_4(request):
+    yield request.param
+
+
+@pytest.mark.llm
+@requires_pt_ge('2.4')
+def test_small_models_toggle_run_args_pt_ge_2_4(
+        caplog, toggle_run_args, small_models_with_ppl_pt_ge_2_4):
+    caplog.set_level(logging.INFO)
+    args = toggle_run_args
+    args.model = small_models_with_ppl_pt_ge_2_4.name
+    exp_float_ppl = small_models_with_ppl_pt_ge_2_4.float_ppl
+    use_fx = requires_fx(args)
+    if use_fx and not small_models_with_ppl_pt_ge_2_4.supports_fx:
+        pytest.xfail(f"{small_models_with_ppl.name} does not support FX")
+    float_ppl, quant_ppl, model = main(args)
+
+
+@pytest.fixture(
+    params=[
+        {
+            "model": "hf-internal-testing/tiny-random-MistralForCausalLM",
+            "act_equalization": "layerwise",
+            "gptq": True,
+            "float_ppl": 31274.05078125,
+            "quant_ppl": 33139.23046875},
+        {
+            "model": "hf-internal-testing/tiny-random-LlamaForCausalLM",
+            "act_equalization": "fx",
+            "bias_corr": True,
+            "float_ppl": 33239.5,
+            "quant_ppl": 33283.75390625},])
+def acc_args_and_acc(default_run_args, request):
+    args = default_run_args
+    run_dict = request.param
+    float_ppl = run_dict["float_ppl"]
+    quant_ppl = run_dict["quant_ppl"]
+    del run_dict["float_ppl"]
+    del run_dict["quant_ppl"]
+    args.update(**run_dict)
+    yield args, float_ppl, quant_ppl
+
+
+@pytest.mark.llm
+@requires_pt_ge('2.2')
```

Comment:
As before, why only 2.2 and up?

Reply:
`optimum-amd[brevitas]` requires PyTorch>=2.2